crates/openapi/src/openapi_v2.rs

```diff
@@ -69,6 +69,11 @@ Never share your secret api keys. Keep them guarded and secure.
     ),
     // The paths will be displayed in the same order as they are registered here
     paths(
+        // Routes for Organization
+        routes::organization::organization_create,
```

Comment:
in order to add these endpoints to mintlify you will have to run this command, run it separately for both v1 and v2 folders

npx @mintlify/scraping@latest openapi-file openapi_spec.json  -o api-reference